ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -84,6 +84,18 @@ internal sealed class ArchiWebHandler : IDisposable {
 			WebBrowser.Dispose();
 		}
 
+		internal async Task<bool> AcceptDigitalGiftCard(ulong gid) {
+			const string requestRedeemGift = "/gifts/0/resolvegiftcard";
+			Steam.NumberResponse result = await UrlPostToJsonObjectWithSession<Steam.NumberResponse>(
```

Comment:
Assign this dictionary to `data` like everywhere else, keep convention, make it single line, it's very unreadable in its current form.